bundles/org.openhab.binding.publictransportswitzerland/src/main/java/org/openhab/binding/publictransportswitzerland/internal/stationboard/PublicTransportSwitzerlandStationboardHandler.java

```diff
@@ -0,0 +1,187 @@
+
+            updateChannels(jsonObject);
+            updateStatus(ThingStatus.ONLINE);
+        } catch (Exception e) {
+            logger.warn("Unable to fetch stationboard data", e);
+
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, e.getMessage());
+            updateState(CHANNEL_TSV, new StringType("No data available"));
+        }
+    }
+
+    private static String createFilterForFields(String... fields) {
+        return Arrays.stream(fields).map((field) -> "&fields[]=" + field).collect(Collectors.joining());
+    }
+
+    private void updateChannels(JsonElement jsonObject) throws Exception {
+        JsonArray stationboard = jsonObject.getAsJsonObject().get("stationboard").getAsJsonArray();
+        createDynamicChannels(stationboard.size());
+
+        List<String> tsvRows = new ArrayList<>();
+
+        for (int i = 0; i < stationboard.size(); i++) {
+            JsonElement jsonElement = stationboard.get(i);
+
+            JsonObject departureObject = jsonElement.getAsJsonObject();
+            JsonObject stopObject = departureObject.get("stop").getAsJsonObject();
+
+            String category = departureObject.get("category").getAsString();
+            String number = departureObject.get("number").getAsString();
+            String destination = departureObject.get("to").getAsString();
+
+            JsonElement delayElement = departureObject.get("delay");
+            String delay = "";
+            if (delayElement != null) {
+                delay = delayElement.getAsString();
+            }
+
+            Long departureTime = stopObject.get("departureTimestamp").getAsLong();
+            String track = stopObject.get("platform").getAsString();
+
+            updateState(getChannelUIDForPosition(i), new StringType(formatDeparture(category, number, departureTime, destination, track, delay)));
+            tsvRows.add(String.join("\t", category, number, departureTime.toString(), destination, track, delay));
+        }
+
+        updateState(CHANNEL_TSV, new StringType(String.join("\n", tsvRows)));
+    }
+
+    private String formatDeparture(String category, String number, Long departureTimestamp, String destination, String track, @Nullable String delay) {
+        Date departureDate = new Date(departureTimestamp * 1000);
+
+        SimpleDateFormat timeFormat = new SimpleDateFormat("HH:mm");
+        String formattedDate = timeFormat.format(departureDate);
+
+        String train = number.startsWith(category) ? category : category + number;
+        String result = String.format("%s - %s %s - Pl. %s", formattedDate, train, destination, track);
+
+        if (delay != null && !delay.isEmpty()) {
+            result += String.format(" (%s' late)", delay);
+        }
+
+        return result;
+    }
+
+    private void createDynamicChannels(int numberOfChannels) {
+        List<Channel> existingChannels = getThing().getChannelsOfGroup(dynamicChannelGroupUID.getId());
```

Comment:
You create the dynamic channels in a group.
But your channel tsv is not in a group.
I have in mind that if a binding is using a group of channels, all channels have to be in a group.
Can you check how does it behave in Paper UI ? Do you get an access to the dynamic channels and your tsv channel ?

Reply:
<img width="573" alt="image" src="https://user-images.githubusercontent.com/7629727/82709021-f7c6c500-9c7f-11ea-88ab-eaec58ee1e29.png">
I get access to all channels

Reply:
Ok, in this case, Paper UI does not display the group.

Reply:
As PaperUI will be removed in OH3, the successor could behave differently. Could you put the TSV Channel in a channel group, too? ["A thing can only have direct channels or channel groups, but not both."](https://www.openhab.org/docs/developer/bindings/thing-xml.html#channel-groups)